tests/test_entities.py

```diff
@@ -22,4 +22,6 @@ def test_entities():
     assert fix_text_segment('ellipsis&#x85;', normalization='NFKC') == 'ellipsis...'
     assert fix_text_segment('broken&#x81;') == 'broken\x81'
     assert unescape_html('euro &#x80;') == 'euro €'
-    assert unescape_html('not an entity &#20x6;') == 'not an entity &#20x6;'
+    assert unescape_html('not an entity &#y20x6;') == 'not an entity &#y20x6;'
+    assert unescape_html('partial entity &#20x6;') == 'partial entity x6;'
+    assert unescape_html('&amp&amp&AMP') == '&&&'
```

Comment:
Just to check: all three of these pass because things are delegated to `html.unescape()`; do there need to be tests here of the other work that's done in `unescape_html` (e.g. the `P&EACUTE;REZ` fix), or do the doctests for that function sufficiently cover it?